server/apps/publish/archive_publish.py

```diff
@@ -142,12 +139,8 @@ def update(self, id, updates, original):
             raise SuperdeskApiError.internalError(message="Failed to publish the item: {}"
                                                   .format(str(e)))
 
-    def publish(self, doc, updates, target_output_channels=None):
-        any_channel_closed = self.queue_transmission(doc=doc, target_output_channels=target_output_channels)
-        task = self.__send_to_publish_stage(doc)
-        if task:
-            updates['task'] = task
-        return any_channel_closed
+    def publish(self, doc, target_output_channels=None):
```

Comment:
Since this is just invoking `queue_transmission` can we remove this method and directly invoke `queue_transmission` above?